manticore/core/cpu/arm.py

```diff
@@ -464,12 +467,26 @@ def _wrap_operands(self, ops):
         return [Armv7Operand(self, op) for op in ops]
 
     def shouldCommitFlags(cpu):
+        #workaround for a capstone bug;
+        #the bug has been fixed the 'master' and 'next' branches of capstone as of 2017-07-31
```

Comment:
Add issue id to comments to ease keeping track of it? And move `return` on next line to its own line?